### ISSUE TITLE
add DT3Empty inside DT3

### ISSUE THREAD

**Issue by adderly** [smellslikedonkey/DT3#5]
Having the empty project within the Engine itself, makes it easier for getting started.
It just needs to be pointed out in a little doc.

**Comment by smellslikedonkey** [comment]
I agree. The way it is now is mostly an artifact of it coming from an SVN repo.

**Comment by adderly** [comment]
ok. I'll merge it later. I am trying to compile the LevelEditor.

**Comment by smellslikedonkey** [comment]
I added you as a contributor to the projects so go for it! Let me know when you're done and I'll delete the DT3Empty repo.
